docs/dev/ide-setup/eclipse.md

```diff
@@ -0,0 +1,49 @@
+# *Eclipse* Setup
+
+##  Plugins:
+  - ***Buildship Gradle Integration***
+     1. [Install Plugin](https://marketplace.eclipse.org/content/buildship-gradle-integration)
+  - ***Checkstyle***
+     1. [Install Plugin](http://eclipse-cs.sourceforge.net)
+     1. Follow: https://checkstyle.org/eclipse-cs/#!/project-setup
+     1. After step 3 "Activate Checkstyle for your project", select the TripleA configuration file
+      instead of "Sun Checks". Checkstyle file: [triplea/config/checkstyle/checkstyle.xml
+     ](https://github.com/triplea-game/triplea/blob/master/config/checkstyle/checkstyle.xml)
+  - ***Lombok*** 
+    1. Download [jar file](https://projectlombok.org/downloads/lombok.jar) and execute it: `java -jar lombok.jar`
+       - [manual install](https://groups.google.com/forum/#!topic/project-lombok/3rVS0eXVl5U)
+         if that does not work
+       - Version to use: Same version as Gradle (see `lombok` configuration in the top-level [Gradle buildscript](https://github.com/triplea-game/triplea/blob/master/build.gradle))
+    2. Enable annotation processing: [how-to-configure-java-annotation-processors-in-eclipse
+      ](https://stackoverflow.com/questions/43404891/how-to-configure-java-annotation-processors-in-eclipse)
+
+## Formatter
+1. [Install Google Java Format Plugin](https://github.com/google/google-java-format#eclipse)
+1. *Project > Java Code Style > Formatter*
+   1. Import profile, select the Triplea formatter file: [triplea/.ide-eclipse/format/triplea-eclipse-java-google-style.xml
+     ](https://github.com/triplea-game/triplea/blob/master/.ide-eclipse/format/triplea-eclipse-java-google-style.xml)
+   1. Change Formatter implementation to 'google-java-format'
+![Screenshot from 2019-08-08 16-46-41
+](https://user-images.githubusercontent.com/12397753/62744601-224ed680-b9fc-11e9-8922-df3564c4207f.png)
```

Comment:
Why those weird line breaks?
AFAIK md files don't have an enforced max line length

Reply:
The breaking of links is pretty consistent, it's mainly to avoid long line wrappings. It's a maintenance problem and hard to read from source when MD files have very long lines. Horizontal scrolling is bad.